docs/blog/2023-08-swarm-0.4-release.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Swarm 0.4 release"
+---
+
+    [BLOpts]
+    profile = wp
+    tags = Swarm, game, robot, programming, resource
+    categories = Haskell, projects
+
+The [Swarm](https://github.com/swarm-game/swarm/) development team is
+very proud to announce the latest release of the game.  This should
+still be considered a development/preview release---you still can't
+[save your games](https://github.com/swarm-game/swarm/issues/50),
+though we're making progress towards that---but it's made some
+remarkable progress and there are lots of fun things to try.  We
+always [welcome any new contributors][contrib]---from playtesting, to
+designing new scenarios, to hacking on the Haskell codebase, all
```

Comment:
Use a "closing" long dash
```suggestion
designing new scenarios, to hacking on the Haskell codebase---all
```

Reply:
I just removed the entire last sentence.  Editing via deleting fluff is always so satisfying.